htdocs/core/lib/functions.lib.php

```diff
@@ -14445,16 +14445,19 @@ function show_actions_messaging($conf, $langs, $db, $filterobj, $objcon = null,
 				if ($truncateLines > 0 && strlen($histo[$key]['message']) > strlen($truncatedText)) {
 					$out .= '<div class="readmore-block --closed" >';
 					$out .= '	<div class="readmore-block__excerpt">';
-					$out .= 	dolPrintHTML($truncatedText);
+					//$out .= 	dolPrintHTML($truncatedText);
+					$out .= 	dol_htmlwithnojs(dol_string_onlythesehtmltags($truncatedText, 1, 1, 1));
 					$out .= ' 	<br><a class="read-more-link" data-read-more-action="open" href="'.DOL_MAIN_URL_ROOT.'/comm/action/card.php?id='.$actionstatic->id.'&backtopage='.urlencode($_SERVER["PHP_SELF"].'?'.$param).'" >'.$langs->trans("ReadMore").' <span class="fa fa-chevron-right" aria-hidden="true"></span></a>';
 					$out .= '	</div>';
 					$out .= '	<div class="readmore-block__full-text" >';
-					$out .=  dolPrintHTML($histo[$key]['message']);
+					//$out .=  dolPrintHTML($histo[$key]['message']);
+					$out .=  dol_htmlwithnojs(dol_string_onlythesehtmltags($histo[$key]['message'], 1, 1, 1));
```

Comment:
We should never have to use a complex combination of escaping / sanitizing function. To avoid this the dolPrintHTML was introduce.
I do not experience trouble on my side. What is the string value you have in $histo[$key]['message'] ?